.github/workflows/macos.yml

```diff
@@ -12,7 +12,7 @@ env:
 
 jobs:
   x86_64-darwin-test:
```

Comment:
Should we make this more generic given this job is now technically _not_ x86_64?